src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -1004,6 +1021,20 @@ public function itShouldBePricedAtForChannel(ProductInterface $product, $price,
     }
 
     /**
+     * @Then /^(its|this products) original price should be (?:€|£|\$)([^"]+) for channel "([^"]+)"$/
```

Comment:
We usually put prices inside quotation marks, so `(?:€|£|\$)([^"]+)` -> `"(?:€|£|\$)([^"]+)"`